wasm-web-service/greeter/src/actions/greeter/say_hello.rs

```diff
@@ -0,0 +1,5 @@
+use crate::actions::greeter_service::say_hello::*;
+
+pub(crate) async fn task(input: Inputs) -> Result<Outputs, crate::Error> {
```

Comment:
Is this idiomatic for Rust or more personal preference? Can the codegen accommodate having operations for an interface in a single file _and_ operation per file? That is simple in Go. In other languages (e.g. Java, C#), the implementation will be forced into a single file.

Reply:
@pkedy it can, but since `apex` can't yet edit existing files it leaves new code to be written by hand when the spec changes, which can be confusing. It's a tradeoff and I'm not 100% on the best approach.